ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -69,6 +73,24 @@
             ctx)
            {:status 200}))))
 
+(defn- add-cmr-ingest-to-context
+  "Add CMR Ingest prefix to token in the request context of the given request.
+   This function should be called on routes that will ingest into CMR.
+   The CMR Ingest prefix is used to indicates to legacy services that when
+   validating the Launchpad token, the NAMS CMR Ingest group should also be checked.
+   Ingest will only be allowed if the user is in the NAMS CMR Ingest group and
+   also has the right ACLs which is based on Earthdata Login uid."
+  [request]
+  (let [token (-> request :request-context :token)]
+    (if (> (count token) URS_TOKEN_MAX_LENGTH)
```

Comment:
Even though it is just one line it might be worth pulling this out into its own function `is-launchpad-token?` to make it obvious this is the check we're performing to decide if a token is a Launchpad token.

Reply:
Good idea. Done.